buildimage.go

```diff
@@ -32,7 +33,11 @@ func main() {
 		{"go", "get", "-u", "github.com/u-root/u-root"},
 		{"go", "get", "-d", "-v", "-u", "github.com/u-root/NiChrome/..."},
 		{"sudo", "apt", "install", "wireless-tools"},
-		{"go", "run", "github.com/u-root/u-root/.", "-files", "/sbin/iwconfig:bin/iwconfig", "-files", "/sbin/iwlist:bin/iwlist", *uroot, *cmds, *wcmds, *ncmds},
+		{"go", "run", "github.com/u-root/u-root/.", "-files", "/sbin/iwconfig:/bin/iwconfig", "-files", "/sbin/iwlist:/bin/iwlist", *uroot, *cmds, *wcmds, *ncmds},
+	}
+	for _, cmd := range commands {
+		debug("Run %v", cmd)
```

Comment:
why is there two for loops ranging over the same thing and performing the same acts ?

Reply:
The other loop came from merging your edits into my branch.I removed the second loop.